cmd/metrics_exporter/metrics_exporter.go

```diff
@@ -175,9 +175,9 @@ var deleteMetricsExporterCmd = &cobra.Command{
 }
 
 var removeMetricsExporterFromClusterCmd = &cobra.Command{
-	Use:   "remove-from-cluster",
-	Short: "Remove Metrics Exporter Config from Cluster",
-	Long:  "Remove Metrics Exporter Config from Cluster",
+	Use:   "unassign",
```

Comment:
Isn't this a breaking change since we already released a version?

Reply:
Well, I agree, I will let @shinji62 take the call. Just one concern, let' say someone installed v0.1.15 and used this command and later they upgraded to v0.1.6, something which worked earlier will stop working and that doesn't look good.

Reply:
Added alias that old command should work